apps/admin-server/src/pages/index.tsx

```diff
@@ -1,11 +1,23 @@
 import { Inter } from 'next/font/google';
 import { signIn, signOut, useSession } from 'next-auth/react';
 import { useRouter } from 'next/navigation';
+import { useEffect } from 'react';
 
 const inter = Inter({ subsets: ['latin'] });
 
 export default function Home() {
   const { data } = useSession();
```

Comment:
Je zou een wrapper hook kunnen maken om daarin dan de useSession hook van next-auth te kunnen doen, en in dezelfde wrapper hook zet je dan ook de useEffect in. De wrapper hook returned dan de {data} van de useSession hook en dan hoef je niet in elke page de useEffect apart op te zetten